presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcTableHandle.java

```diff
@@ -47,12 +49,14 @@ public JdbcTableHandle(
             @JsonProperty("catalogName") @Nullable String catalogName,
             @JsonProperty("schemaName") @Nullable String schemaName,
             @JsonProperty("tableName") String tableName,
+            @JsonProperty("limit") OptionalLong limit,
```

Comment:
I'd add this after `domain`, because logically, limit is applied later.

(applies to ctor parameters, fields, ctor init order, getter)

Reply:
I did this to make the diff cleaner, but you're right, I'll change it.